src/client-agent/start_agent.c

```diff
@@ -402,3 +403,24 @@ void send_agent_stopped_message() {
     /* Send shutdown message */
     send_msg(msg, -1);
 }
+
+/**
+ * @brief Resolve manager hostnames
+ */
+void agent_setup_hostnames() {
+    for (int rc = 0; rc < agt->server_count; rc++) {
+        if (OS_IsValidIP(agt->server[rc].rip, NULL) != 1) {
+            mdebug2("Resolving server hostname: %s", agt->server[rc].rip);
+            resolve_hostname(&agt->server[rc].rip, 5);
+            int rip_l = strlen(agt->server[rc].rip);
+
+            if (agt->server[rc].rip[rip_l - 1] == '/') {
+                mwarn("Could not resolve server hostname: %.*s", rip_l - 1, agt->server[rc].rip);
```

Comment:
@palaciosjeremias thank you for all your comments.

In this case, as privately told, I prefer the user to get a warning because the most common cause of this log is a typo in the manager address. These alternatives seem weird to me:

```
INFO: Could not resolve server hostname: unknown
INFO: Server hostname resolved: unknown/
```